app/graphql/mutations/webhooks/resend.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+module Mutations
+  module Webhooks
+    class Resend < BaseMutation
```

Comment:
What about changing the naming from `Resend` to `Retry` eveyrwhere? 😬 

If I understand correctly, nothing has already been sent as it was a failure. Plus it can be consistent with the naming of `last_retried_at` column and the variable `LAGO_WEBHOOK_ATTEMPTS`.

Reply:
we can but I'll keep the mutation name as `ResendWebhook` if that works for you